### DISCUSSION TITLE
Usage of ViewAnalysis

### DISCUSSION THREAD

**Discussion by nemery-flextrade** [finos/FDC3#602] [Q&A] [answered]
We currently have a ViewAnalysis intent, I was wondering what the use cases for this were?

I just thought about a use case for something along the lines of viewing pre-trade analytics and wasn't sure if that would come under it's own intent or be bundled under ViewAnalysis?

**Comment by kriswest** [comment, marked as answer]
The current description is the rather basic:
> Display analysis on the provided context.

I think the use case you mention fits squarely under that.

However, given that the existing description didn't answer the question well, suggesting an improved description, use-cases etc. would be great.

**Reply by nemery-flextrade** [reply]
Thanks Kris. Let me come up with something and run it past some of the more business-y folk internally and I'll raise an issue.
